lib/process-services-cloud/src/lib/form/components/widgets/date/date-cloud.widget.html

```diff
@@ -1,11 +1,11 @@
 <div class="{{field.className}}" id="data-widget" [class.adf-invalid]="!field.isValid && isTouched()" [class.adf-left-label-input-container]="field.leftLabels">
     <div *ngIf="field.leftLabels">
-        <label class="adf-label adf-left-label" [attr.for]="field.id">{{field.name | translate }} ({{field.dateDisplayFormat}})<span class="adf-asterisk"
+        <label class="adf-label adf-left-label" [attr.for]="field.id">{{field.name | translate }} ({{dateFormatTranslationService.convertDateFnsToMomentFormat(field.dateDisplayFormat)}})<span class="adf-asterisk"
```

Comment:
this needs to be extracted to a function, UI templates should not deal with services directly, see https://github.com/Alfresco/alfresco-ng2-components/pull/8924/files#r1335592062